### PR TITLE
Add TabletService to the public api

### PR DESCRIPTION
**Pull request by snaury**
### Changelog entry 

...

### Changelog category 

* Not for changelog (changelog entry is not required)

### Additional information

...

### CODE REVIEW

File: ydb/public/api/grpc/ydb_tablet_v1.proto [hunk 1]
```diff
@@ -0,0 +1,24 @@
+syntax = "proto3";
+
+package Ydb.Tablet.V1;
+option java_package = "com.yandex.ydb.tablet.v1";
+
+import "ydb/public/api/protos/ydb_tablet.proto";
+
+service TabletService {
```
Comment (fomichev3000) [inline ydb/public/api/grpc/ydb_tablet_v1.proto:8]: I recommend adding a comment, that this service is a very low entity and is used for database hacking.

File: ydb/public/api/grpc/ydb_tablet_v1.proto [hunk 2]
```diff
@@ -0,0 +1,24 @@
+syntax = "proto3";
```
Comment (fomichev3000) [inline ydb/public/api/grpc/ydb_tablet_v1.proto:1]: I suppose it's better to move this file into draft folder.

**Review by fomichev3000 — CHANGES_REQUESTED** [review]

File: ydb/public/api/grpc/ydb_tablet_v1.proto (hunk 1 above)
Comment (snaury) [reply ydb/public/api/grpc/ydb_tablet_v1.proto]: Done

File: ydb/public/api/grpc/ydb_tablet_v1.proto (hunk 2 above)
Comment (snaury) [reply ydb/public/api/grpc/ydb_tablet_v1.proto]: Done

**Review by fomichev3000 — APPROVED** [review]

**Pull request merged**